addons/web/static/src/core/position_hook.js

```diff
@@ -83,7 +83,9 @@ const DEFAULTS = {
  * @param {HTMLElement} reference
  * @param {HTMLElement} popper
  * @param {Options} options
- * @returns {PositioningSolution} the best positioning solution
+ * @returns {PositioningSolution} the best positioning solution, relative to
+ *                                the containing block of the popper.
+ *                                => can be applied to popper.style.(top|left)
```

Comment:
not sure we need to update the JSDoc for a non exported function, but why not ;-)